source/content/guides/decoupled-sites/03-starter-kits.md

```diff
@@ -16,35 +16,60 @@ reviewed: "2022-08-11"
 
 ## Create the CMS Backend
 
-You can create a decoupled site using a Pantheon starter kit. Choose from either a WordPress or Drupal CMS and connect it to the frontend application to create a site. The following configurations exist for the starter kit templates:
+You can create a Front-End Site by configuring your Dashboard upstream or by using a Pantheon starter kit. Choose either a WordPress or Drupal CMS and connect it to the frontend application to create a site. The following configurations exist for the starter kit templates:
```

Comment:
This kind of mixes concepts from CMS sites and Front-end sites in a way that I think could be confusing. Would this still work?

You can create a Front-End Site by using a Pantheon starter kit or importing from an existing repository.